resources/google-music-electron-old.svg

```diff
@@ -0,0 +1,488 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Ah, yea. We prob don't need this in the repo since we are sticking with the other version. Feel free to throw it in a gist and link to it on this issue though